service/create/policy.go

```diff
@@ -102,3 +116,69 @@ func createInstanceProfile(svc *iam.IAM, clusterID string) (string, error) {
 
 	return clusterProfileName, nil
 }
+
+func deleteInstanceProfile(svc *iam.IAM, clusterID string) error {
+	clusterProfileName := fmt.Sprintf("%s-%s", clusterID, ProfileNameTemplate)
+
+	_, err := svc.DeleteInstanceProfile(&iam.DeleteInstanceProfileInput{
+		InstanceProfileName: aws.String(clusterProfileName),
+	})
+
```

Comment:
Cosmetic. I personally would like to not have empty lines between errors and conditional statements.